_posts/2020-06-07-higher-order-hydras.markdown

```diff
@@ -0,0 +1,491 @@
+This ordering is a well-order with order type whatever the hydra system's ordinal is.
+Amiko hydras follow this too, and they do correspond to some large ordinals.
+
+However, even if Amiko hydras themselves are already an ordinal notation, this is not quite complete.
+If we look at Buchholz hydras, the hydras can be converted into expressions using Buchholz's \\\(\psi\\\), which is an ordinal collapsing function defined just using ordinals.
+I have not yet found or created such a system for Amiko hydras, an ordinal notation defined purely using ordinals which is able to exactly mirror the structure of Amiko hydras.
+It seems challenging, and perhaps that is an item best left for future research.
+
+</div>
+
+# Flattening of Amiko hydras
+{: #amiko-hydras-flattening}
+
+{% include collapser.markdown %}
+
+<div>
+
+For some purposes, it may be useful to flatten the hydra.
+If hydra labels are "internal" and children are "external", in the process of deriving Amiko hydras from the nested hydra system, we externalized that nesting already.
+We can also externalize the hydra labels in Amiko hydras by moving the hydra from the label into a fake first child, except for the root (and sub-roots) which will have no label.
+Rules would need to be slightly reworded so rules about labels instead refer to the first child and rules about children ignore the first child, but the hydras would be functionally the same, just now in tree form without further nesting.
+If you wanted to flatten it further, you could use any of a number of methods which represents a tree uniquely using a list.
+
+</div>
+
+# Fast-growing function and reference number
+{: #amiko-hydras-fast-growing}
+
+{% include collapser.markdown %}
+
+<div>
+
+## Summary of the fast-growing function and reference number
+{: #amiko-hydras-fast-growing}
+
+<div>
+
+For completeness, and for fun, I should define a fast-growing function based on Amiko hydras.
+So, let's define \\\(AH(k)\\\).
+
+Begin with the smallest possible hydra, which is \\\((\star:)\\\).
+Repeat \\\(k\\\) times: \\\(A \mapsto (\star:(A:))\\\).
+This is now the starting hydra.
+
+Begin with a global shared counter \\\(N = 1\\\), which will increment after every time it is called for (including additional mentions when reducing nested hydras), not on each outer evaluation of \\\(S\\\).
+Reduce the hydra using \\\(S\\\) until it is \\\((\star:)\\\).
+The final value of \\\(N\\\) is \\\(AH(k)\\\).
+
+As examples, here are the first few values of \\\(AH\\\):
+
+> \\\(AH(0) = 1\\\)
+> \\\(AH(1) = 1\\\)
+> \\\(AH(2) = 3\\\)
+
+It only makes sense to submit one number to the ever-growing list of large numbers humans have described, since other numbers will likely be far below it or so far above it that if you replaced \\\(f_0\\\) in the fast-growing hierarchy with \\\(AH\\\) then \\\(f_\alpha(N)\\\) for reasonable ordinal \\\(\alpha\\\) and integer \\\(N\\\) it wouldn't even get close to the other number.
```

Comment:
You might want to break up this run-on sentence; it's not too long but splitting it could help readability.

Reply:
It's not a run-on sentence, and I think these clauses should stay joined together if possible for clarity. Commas may help readability but it's also risky because what level the comma separates at is ambiguous. Do you have a suggestion as to what to change it to?

Reply:
lol I think I'm seeing a pattern in myself to deem any sentence that's long enough a run-on sentence 🤣 I'd say keep this then 👍